git.yazi/main.lua

```diff
@@ -204,10 +213,13 @@ local function fetch(_, job)
 	end
 	ya.dict_merge(changed, propagate_down(excluded, cwd, Url(repo)))
 
+	-- make sure the status changed when a file is reverted from a modified state to an unmodified state
+	-- (when we just open the editor from yazi, edit, then back to yazi)
```

Comment:
```suggestion
	-- Reset the state of any files that don't appear in the output of `git status` to `unknown`, so that cleaning up outdated statuses from `st.repos`
```